src/material/tabs/paginated-tab-header.ts

```diff
@@ -209,9 +214,14 @@ export abstract class MatPaginatedTabHeader
 
     // Defer the first call in order to allow for slower browsers to lay out the elements.
     // This helps in cases where the user lands directly on a page with paginated tabs.
-    // Note that we use `onStable` instead of `requestAnimationFrame`, because the latter
-    // can hold up tests that are in a background tab.
-    this._ngZone.onStable.pipe(take(1)).subscribe(realign);
+    // TODO(mmalerba): Consider breaking this into multiple `afterNextRender` calls with explicit
+    //  phase.
+    afterNextRender(
+      () => {
```

Comment:
Should be able to just do `afterNextRender(realign)` without wrapping it in another callback.

Reply:
I kind of like the wrapper because it makes it clear that there's no arguments being passed, but that's more of a personal style preference that I don't feel too strongly about, so changed